infra/discovery/src/listener/handler_proxy.js

```diff
@@ -37,6 +39,12 @@ class ProxyEventHandler {
 
     // Get the address of the newly created proxy from the event.
     const proxyAddress = event.returnValues.proxy
+    if (proxyAddress === ZeroAddress) {
+      // We had a bug causing some transactions to have a zero proxy address.
+      // Skip those...
+      logger.info('ProxyEventHandler: skipping event with zero proxy address.')
```

Comment:
Could maybe be a warning since it's technically an error state, but not a deal breaker.

Reply:
Yes good point. Let me update...